src/BlockTypes/ClassicTemplate.php

```diff
@@ -47,10 +48,17 @@ protected function initialize() {
 	public function enqueue_block_assets() {
 		// Ensures frontend styles for blocks exist in the site editor iframe.
 		if ( class_exists( 'WC_Frontend_Scripts' ) && is_admin() ) {
-			$frontend_scripts = new \WC_Frontend_Scripts();
+			$frontend_scripts = new WC_Frontend_Scripts();
 			$styles           = $frontend_scripts::get_styles();
+
 			foreach ( $styles as $handle => $style ) {
-				wp_enqueue_style( $handle, $style['src'], $style['deps'], $style['version'], $style['media'] );
+				wp_enqueue_style(
+					$handle,
+					substr( $style['src'], 0, 2 ) === '//' ? 'https:' . $style['src'] : $style['src'],
```

Comment:
@mikejolley, shouldn't we also handle the case of no secure websites (HTTP)? This worked fine on my Jurassic Ninja test site but not on my local development site. Maybe there is a solution to include both protocols, or we can do a check as follows:

```PHP
$protocol = strtolower( substr( $_SERVER[ "SERVER_PROTOCOL" ], 0, 5 ) ) == 'https' ? 'https' : 'http';
```

I've tested with `$_SERVER['HTTPS']`, but it doesn't return the correct values on my local server.

Reply:
@tarhi-saad I added `set_url_scheme`. That takes care of it for us.